petitioner_app/views.py

```diff
@@ -28,19 +29,21 @@ def vehregister(request):
             output_document = "output1.docx"
             libre_office_path = 'C:\\Program Files\\LibreOffice\\program\\soffice'
             # Create docx file due to template
+            """
             document = MailMerge(template)
             document.merge_pages([form_dict])
             document.write(output_document)
             document.close()
             # Convert docx to pdf (using libreoffice)
             args = [libre_office_path, '--headless', '--convert-to', 'pdf', output_document]
             subprocess.run(args)
+            """
```

Comment:
ten zakomentowany kawałek był odpowiedziany za stworzenie plików docx i pdf

Reply:
Ta, nie chciałem, żeby mi się odpalało jak tworzę do bazy formularze, nie mam libre. Poza tym docelowow to będzie trzeba i tak przenieść na przyciski u petitionera i clerka

Reply:
Jeśli chcesz, to w następnym commicie odkomentuję, zaraz dorzucam działający widok pojedynczego formularza i wtedy robię widk dla urzędnika.

Reply:
W sumie to bez różnicy. Tylko chciałem zaznaczyć że nie jest to do wywalenia xD